Makefile

```diff
@@ -52,7 +52,7 @@ $(BLS512_LIB): $(OBJ_DIR)/bls_c512.o
 $(BLS384_256_LIB): $(OBJ_DIR)/bls_c384_256.o
 	$(AR) $@ $<
 
-ifneq ($(findstring $(OS),mac/mingw64),)
+ifneq ($(findstring $(OS),mac/mac-m1/mingw64),)
```

Comment:
will this not break the non-m1 mac case?

Reply:
This is including mac-m1 with the normal mac path

Reply:
It should still work like usual with non m1